HopsanGUI/HcomHandler.cpp

```diff
@@ -4718,6 +4726,40 @@ void HcomHandler::executeConnectCommand(const QString cmd)
 }
 
 
+//! @brief Execute function for "unco" command
+void HcomHandler::executeListUnconnectedCommand(const QString cmd)
+{
+    QStringList args = splitCommandArguments(cmd);
+    QString wildcard = "*";
+    if(args.size() == 1)
+    {
+        wildcard = args[0];
+    }
+    else if(args.size() > 1)
+    {
+        HCOMERR("Wrong number of arguments");
+        return;
+    }
+
+    QList<Port*> ports;
+    getPorts(wildcard,ports);
+
+    QStringList portNames;
+    for(Port* pPort : ports)
+    {
+        if(!pPort->isConnected())
+        {
```

Comment:
Maybe we could print the name directly here instead of temp storing in portNames